.github/test/test-workflows.js

```
@@ -0,0 +1,193 @@
+
+const assert = require('node:assert');
+const fs = require('node:fs');
+const path = require('node:path');
+const yaml = require('js-yaml');
+
+// Install js-yaml if not present
+try {
+  require.resolve('js-yaml');
+} catch (e) {
+  console.log('Installing js-yaml for workflow parsing...');
+  require('child_process').execSync('npm install js-yaml', { stdio: 'inherit' });
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)